package.json

```diff
@@ -20,24 +20,27 @@
     "microfrontends",
     "single-spa"
   ],
+  "lint-staged": {
+    "*.*": "prettier --write"
+  },
   "author": "Joel Denning",
   "license": "MIT",
   "devDependencies": {
-    "@babel/core": "^7.20.12",
-    "@babel/eslint-parser": "^7.19.1",
-    "@changesets/changelog-github": "^0.4.8",
-    "@changesets/cli": "^2.26.0",
-    "@vercel/ncc": "^0.36.1",
-    "eslint": "^8.32.0",
+    "@babel/core": "^7.23.7",
+    "@babel/eslint-parser": "^7.23.3",
+    "@changesets/changelog-github": "^0.5.0",
+    "@changesets/cli": "^2.27.1",
+    "@vercel/ncc": "^0.38.1",
+    "eslint": "^8.56.0",
     "eslint-config-important-stuff": "^1.1.0",
     "husky": "^8.0.3",
-    "prettier": "^2.8.3",
-    "pretty-quick": "^3.1.3",
-    "replace-in-file": "^6.3.5",
-    "rimraf": "^4.1.2"
+    "lint-staged": "^15.2.0",
+    "prettier": "^3.2.4",
+    "replace-in-file": "^7.1.0",
+    "rimraf": "^5.0.5"
   },
   "dependencies": {
-    "@actions/core": "^1.10.0",
-    "@baseplate-sdk/cli": "^1.1.4"
+    "@actions/core": "^1.10.1",
+    "@baseplate-sdk/cli": "^2.0.0"
```

Comment:
This is necessary for the autoVersion feature which I'll implement in an upcoming PR